### ISSUE TITLE
FirebaseAnalytics.setConsent KTX references proguarded ConsentBuilder

### ISSUE THREAD

**Issue by hameno** [firebase/firebase-android-sdk#2281]
### [REQUIRED] Step 2: Describe your environment

  * Android Studio version: 4.1.1
  * Firebase Component: Analytics KTX
  * Component version: 18.0.0

#### Steps to reproduce:

Try to use setConsent extensions function on FirebaseAnalytics. Compilation fails because ConsentBuilder is proguarded to zza.

#### Relevant Code:

```kotlin
firebaseAnalytics.setConsent {
    analyticsStorage = FirebaseAnalytics.ConsentStatus.GRANTED
}
```

![image](https://user-images.githubusercontent.com/365084/103011661-9b3fc700-453a-11eb-9ccd-b7e4d53e05e0.png)
![image](https://user-images.githubusercontent.com/365084/103011734-b27eb480-453a-11eb-9c18-9fb3af79f583.png)

**Comment by rlazo** [comment]
HI @hameno thanks for your report! I'll work on a fix and keep you posted of any updates.

**Comment by rlazo** [comment]
Hi @hameno today we released version 18.0.2 which includes the fix. Thanks for your report!
